src/NetworkPort.php

```diff
@@ -293,6 +293,9 @@ public function post_updateItem($history = true)
         $this->updateMetrics();
     }
 
+    /**
+     * @param NetworkPort $source
+     */
```

Comment:
IMHO, the best way to handle this is to use `/** @template T of CommonDBTM */` in the `Clonable` trait with `/** @param T $source */` and remove the PHPDoc here (and maybe add the `Override` attribute).